bookdown/02-Data_Acquisition_Locations.Rmd

```diff
@@ -0,0 +1,151 @@
+lakes within the conterminous US and US Territories using the
+`nhdplusTools::download_nhdplushr()` function [@blodgett2023]. All HUC4s were
+downloaded and processed on
+`r date(ymd_hms(file.info("../a_Calculate_Centers/mid/poi_centers_huc4_0101.csv")$mtime))`
+using the most updated version available at the time of download. Because the
+NHDPlusHR dataset is incomplete for Alaska (Figure \@ref(fig:NHDPlusHR-status)),
+we used the NHD Best Resolution File for the state of AK [@usgeologicalsurvey].
+
+```{r NHDPlusHR-status, echo=F, fig.cap="NHDPlusHR dataset availability, courtesy of the USGS."}
+knitr::include_graphics("https://d9-wret.s3.us-west-2.amazonaws.com/assets/palladium/production/s3fs-public/media/images/NHDPlusHRStatus_Web_20220707_0.png")
+```
+
+For every HUC4 included in the NHDPlusHR dataset and the state of Alaska NHD
+Best Resolution dataset, all waterbodies are limited to those with
+[FCodes](https://files.hawaii.gov/dbedt/op/gis/data/NHD%20Complete%20FCode%20Attribute%20Value%20List.pdf)
+belonging to the following groups: 390 (lake/pond), 436 (reservoir), and 361
+(playa) and are at least 0.01 km^2^ (1 hectare) in area according to the area
+value provided in the NHD file. This filtering resulted in
+`r format(nrow(combined_poi_points), big.mark = ",")` waterbodies included in
+our dataset, including `r format(nrow(AK_poi_points), big.mark = ",")` from
+Alaska.
+
+For each waterbody, the POI and distance-to-shore radius was calculated using
+the `polylabelr::poi()` function. In order to accurately calculate
+distance-to-shore, each waterbody was converted to the proper Universal
+Transverse Mercator (UTM) projection calculated from the mean longitudinal value
+of the polygon vertices prior to applying the `poi()` function. By using the
+point-local UTM projection, we decrease distortion expected from any single CRS
+used to represent all of the locations from which we have lakes. The latitude
+and longitude values of the POI were transformed to decimal degrees in World
+Geodetic System 1984 (WGS84) (EPSG:4326) from UTM easting and northing
+coordinates for use later in the workflow. It is
+important to note that the `poi()` function does not perform particularly well
+on larger polygons, however the points calculated should be an acceptable proxy
+for pelagic lake conditions. [[Will add additional detail from analysis of
+buffer sizes.]]
+
+## Implementation
+
+The code for gathering NHD waterbodies and calculating their respective POIs is
+found in {targets} group *a_Calculate_Centers* which is located in the script
+[a_Calculate_Centers.R](https://github.com/steeleb/lakeSR/blob/main/a_Calculate_Centers.R).
+The network graph (Figure \@ref(fig:a-group-vis)) for this group depicts the
+dependencies between each of the *targets* in this group.
+
+```{r a-group-vis, fig.cap= "Network graph of the *targets* in the a_Calculate_Centers {targets} group."}
```

Comment:
Maybe add message = F?

Reply:
It's set in global at the top, but I see that {targets} is being dumb in the .html version that I sent you.... It does that *sometimes* but not all the time. So strange. I'll re-knit and see if that removes it before I merge!